mLRS/CommonTx/cli.h

```diff
@@ -683,6 +714,14 @@ bool rx_param_changed;
 
     //puts(".");
 
+    if (output_pending == CLI_OUTPUT_HELP) {
```

Comment:
maybe a switch

Reply:
Yes, maybe; especially if we had more states.